envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadata.kt

```diff
@@ -226,29 +274,20 @@ data class Incoming(
 }
 
 data class Outgoing(
-    val dependencies: List<Dependency> = emptyList()
+    val serviceDependencies: List<ServiceDependency> = emptyList(),
+    val domainDependencies: List<DomainDependency> = emptyList(),
+    val allServicesDependencies: Boolean = false,
+    val defaultServiceSettings: DependencySettings = DependencySettings()
 ) {
-    fun containsDependencyForService(service: String) = serviceDependencies.containsKey(service)
-
-    // not declared in primary constructor to exclude from equals(), copy(), etc.
```

Comment:
Why is this no longer done this way?

Reply:
Does equals still work the same? Did snapshot rate change?